nemo/utils/export_utils.py

```diff
@@ -219,7 +277,4 @@ def replace_for_export(model: nn.Module, replace_1D_2D: bool = False) -> nn.Modu
     Returns:
         model, possibly modified in-place
     """
-    replace_modules(model, default_Apex_replacements)
-    if replace_1D_2D:
-        # TODO: add squeeze/unsqueeze
-        replace_modules(model, default_1D_2D_replacements)
+    replace_modules(model, default_replacements)
```

Comment:
This will replace every instance of BN1d - even for citrinet, ContextNet, Jasper QN etc - will they be ok with this chance ?
Is this some regression in onnx itself for BN1d?

Reply:
Yes they will be OK and that's what we want. BN1d weights are always in fp32, so those casts are no-ops, really - just helping to generate proper ONNX code.